source/enterprise/tech-guide/update-eca-guide.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: Update the Client Administration
+github_link: enterprise/tech-guide/update-eca-guide.md
+indexed: false
+---
+
+This guide describes how you update the Enterprise Client Administration.
+
+<div class="toc-list"></div>
+
+## How to update the ECA
+
+----------------------
+
+### 1. Start the maintenance mode
+
+First of all, you should turn the eca into the maintenance mode for security reasons.
+This blocks the frontend from every user interaction.
+
+For this just create a app.lock file in the root of your eca directory.
```

Comment:
To enable the maintenance mode create a `app.lock` file in the root directory of your eca.